packages/squashfs.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Squashfs < Package
+  description 'Squashfs is a compressed read-only filesystem for Linux.'
+  homepage 'http://squashfs.sourceforge.net/'
+  version '4.3'
+  source_url 'https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs4.3/squashfs4.3.tar.gz'
+  source_sha256 '0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'help2man' => :build
+
+  def self.build
+    system "cd squashfs-tools && make"
+  end
+
+  def self.install
+    FileUtils.cd('squashfs-tools') do
+      system "mkdir -p #{CREW_DEST_PREFIX}/share/man/man1"
+      system "install -Dm755 mksquashfs #{CREW_DEST_PREFIX}/bin/mksquashfs"
+      system "install -Dm755 unsquashfs #{CREW_DEST_PREFIX}/bin/unsquashfs"
+      system "help2man ./mksquashfs -N --no-discard-stderr > #{CREW_DEST_PREFIX}/share/man/man1/mksquashfs.1"
```

Comment:
Does help2man have an option to produce a compressed manpage?

Reply:
No, but I could use compressdoc.  I'll work on it.